src/js/claims-status/components/appeals-v2/Timeline.jsx

```diff
@@ -23,7 +23,10 @@ class Timeline extends React.Component {
     return `${first} - ${last}`;
   };
 
-  toggleExpanded = () => this.setState((prevState) => ({ expanded: !prevState.expanded }));
+  toggleExpanded = (e) => {
+    e.stopPropagation();
+    this.setState((prevState) => ({ expanded: !prevState.expanded }));
```

Comment:
Interesting, what's the advantage of using `prevState` over `this.state`?

Reply:
:thinking: Y'know, I don't remember....I'm sure I had a good reason!